<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>320</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="mainStackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>631</width>
      <height>311</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <widget class="QWidget" name="welcomePage">
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>30</y>
        <width>481</width>
        <height>20</height>
       </rect>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="text">
       <string>Welcome to Volcabulator</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>80</y>
        <width>201</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>Select file with volcabulary:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnSelectVocabularyFile">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>80</y>
        <width>87</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string>Browse</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelFileSelected">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>80</y>
        <width>301</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>No file selected</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnPractice">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>170</y>
        <width>171</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Practice</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnModifyVocabularyFile">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>170</y>
        <width>171</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Modify</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelError">
      <property name="geometry">
       <rect>
        <x>85</x>
        <y>260</y>
        <width>431</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="volcabularyTrainer">
     <widget class="QPushButton" name="btnCheckTranslation">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>130</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Zkontrolovat</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditTranslation">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>130</y>
        <width>281</width>
        <height>31</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>false</bool>
      </property>
      <property name="placeholderText">
       <string>Zadej překlad</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelResult">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>190</y>
        <width>491</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="labelWordToTranslate">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>30</y>
        <width>331</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="btnNextTranslation">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>240</y>
        <width>151</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Další &gt;&gt;</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
